pkg/features/pilot/pilot.go

```diff
@@ -72,6 +72,13 @@ var (
 	// Default is 10s, Example: "300ms", "10s" or "2h45m".
 	DebounceMax = env.RegisterDurationVar("PILOT_DEBOUNCE_MAX", 10*time.Second, "").Get()
 
+	EnableEDSDebounce = env.RegisterBoolVar(
+		"PILOT_ENABLE_EDS_DEBOUNCE",
+		true,
+		"If enabled, Pilot will include EDS pushes in the push debouncing, configured by PILOT_DEBOUNCE_AFTER and PILOT_DEBOUNCE_MAX."+
+			" EDS pushes may be delayed, but there will be fewer pushes. By default this is enabled",
+	)
```

Comment:
add` .Get() `here to keep consistent as other flags

Reply:
This is left to later in the code so it can be tested. Some other flags do this as well